documentation/index.md

```diff
@@ -2,35 +2,33 @@
 
 Openverse is a search engine for openly licensed media.
 
-## Documentation
+# Documentation
```

Comment:
@sarayourfriend isn't this the primary h1 for the page?

Reply:
It could be. They were all H2s and I was just trying to fix errors that already existed and honestly did not read the headings: I assumed the original equal-level was the intention of the author. Reading them now though it does seem like it could be the H1 and the others sub headings.